toolchain/check/testdata/class/generic/adapt.carbon

```diff
@@ -53,33 +46,45 @@ class Adapter {
   extend adapt C(i32);
 }
 
-fn Access(a: Adapter) -> C(i32) {
+fn Access(a: Adapter) -> i32 {
   // TODO: This should presumably work, but the design doesn't say how yet.
-  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+13]]:3: error: cannot implicitly convert from `i32` to `C(i32)` [ImplicitAsConversionFailure]
-  // CHECK:STDERR:   return a.x;
-  // CHECK:STDERR:   ^~~~~~~~~~~
-  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+10]]:3: note: type `i32` does not implement interface `ImplicitAs(C(i32))` [MissingImplInMemberAccessNote]
-  // CHECK:STDERR:   return a.x;
-  // CHECK:STDERR:   ^~~~~~~~~~~
-  // CHECK:STDERR:
-  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+6]]:10: error: cannot implicitly convert from `Adapter` to `C(i32)` [ImplicitAsConversionFailure]
+  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+7]]:10: error: cannot implicitly convert from `Adapter` to `C(i32)` [ImplicitAsConversionFailure]
   // CHECK:STDERR:   return a.x;
   // CHECK:STDERR:          ^~~
-  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+3]]:10: note: type `Adapter` does not implement interface `ImplicitAs(C(i32))` [MissingImplInMemberAccessNote]
+  // CHECK:STDERR: fail_todo_extend_adapt_specific_type.carbon:[[@LINE+4]]:10: note: type `Adapter` does not implement interface `ImplicitAs(C(i32))` [MissingImplInMemberAccessNote]
   // CHECK:STDERR:   return a.x;
   // CHECK:STDERR:          ^~~
+  // CHECK:STDERR:
   return a.x;
 }
 
+// --- extend_adapt_specific_type_copy.carbon
```

Comment:
Consider renaming this to something like `extend_adapt_specific_type_library.carbon`, and renaming the previous file to `fail_todo_local_extend_adapt_specific_type.carbon`. That makes the intent behind the duplication a little clearer.